src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -106,6 +107,10 @@ class SpanDetail extends React.Component<Props, State> {
       end,
     };
 
+    if (!query.project) {
```

Comment:
Maybe collapse the above with this to reduce 2 conditionals to one?
```js
const query = {
  ...,
  query: ...,
  start: ...,
  ...
};

if (!organization.features.includes('global-views')) {
  query.project = [Number(event.projectID)];
}

Reply:
@k-fish typescript doesnt like this form.

Reply:
riiight, forgot about that. Okay 🤷

Reply:
but this is an alternative https://github.com/getsentry/sentry/pull/19635/commits/39d8b2bc499192cc7e97d79f8c43f9de27793317